src/pkg/packaging-tools/packaging-tools.targets

```diff
@@ -147,9 +147,15 @@
     </MSBuild>
 
     <ItemGroup>
+      <!--
+        Add the platform manifest, if it exists. It may not exist if this is, for example, an x64
+        GetFilesToPackage build running under TargetArchitecture=x86.
+      -->
```

Comment:
Do you also want to ensure you get at least one?

Reply:
Yeah... not really happy with this fix. I'm looking if I can make this target only run where it should rather than making it flexible. (If I can find the right place to put the check to make sure one's found, then I imagine I can just get this target to run in that situation.)

Reply:
I think I misunderstood, I need to clarify--this target only ever expects to find 1 or 0, the issue is that it's finding the place a platform manifest *would* exist if the other arch were built. This exists check would make sure it doesn't try to use a platform manifest that wasn't built, but it runs the risk of not erroring out if this target fails to find 1 when it should have.

Reply:
I see, I had expected this to only run 1:1 on the cases where we needed to locate the platform manifest.

Reply:
Realized I was looking at this the wrong way: the platform manifest is for all platforms, not just x86 or x64 in particular, so the path was wrong and 1 is always expected even with the broad `BeforeTargets="ResolveProjectReferences"`. Fixing it up in this direction (and some related issues I iterated on to get the sfx and runtime pack how we expect).